src/Shell/Task/ModelTask.php

```diff
@@ -104,37 +104,51 @@ public function main($name = null)
      */
     public function bake($name)
     {
+        extract($this->getTableObjectSetup($name)); // $tableObject, $data
```

Comment:
Can we use list() instead of extract(). extract() has the downside of making the current function's scope totally opaque.